.github/pull_request_template.md

```diff
@@ -0,0 +1,27 @@
+Fixes #(issue) *(if applicable)*
+
+**Description**
+
+Please include a summary of the change.
+
+**Schema** *(if applicable)*
+
+```jsx
+```
+
+**Checklist:** *(please see [documentation page](https://data-driven-forms.org/development-setup) for more information)*
+
+- [ ] `Yarn build` passes
+- [ ] `Yarn lint` passes
+- [ ] `Yarn test` passes
+- [ ] Test coverage for new code *(if applicable)*
+- [ ] Documentation update *(if applicable)*
+- [ ] Correct commit message
+   - format `fix|feat({scope}): {description}`
+   - i.e. `fix(pf3): wizard correctly handles next button`
+   - fix will release a new x.x.X version
```

Comment:
I would change this to *.*.X or something. The upper/lower case difference is not enough for my eyes.